sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/LocalCryptographyProvider.cs

```diff
@@ -92,17 +92,23 @@ public virtual Task<WrapResult> WrapKeyAsync(KeyWrapAlgorithm algorithm, byte[]
             return Task.FromResult(result);
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static NotSupportedException ThrowOperationNotSupported(string name) => new NotSupportedException($"Operation {name} not supported with the given key");
```

Comment:
```suggestion
        internal static NotSupportedException CreateOperationNotSupported(string name) => new NotSupportedException($"Operation {name} not supported with the given key");
```

Reply:
Will do. Not committing change, though, since all references have to change (easier to just do that in VS).